playbooks/generic/facts.yml

```diff
@@ -9,4 +9,6 @@
     - role: osism.commons.facts
 
 - name: Import gather-facts play
-  ansible.builtin.import_playbook: gather-facts.yml
+  hosts: "{{ hosts_facts|default(hosts_default_group|default('generic')) }}"
```

Comment:
Use all as default value. We want to gather facts from all known hosts by default.

Reply:
Done.